app/javascript/pages/listings/listing-detail.tsx

```diff
@@ -102,7 +102,19 @@ const ListingDetail = () => {
       >
         <aside className="w-full static md:absolute md:right-0 md:w-1/3 md:top-0 sm:w-2/3 md:ml-2 h-full md:border border-solid bg-white">
           <div className="hidden md:block">
-            <ApplicationStatus {...getListingImageCardStatuses(listing, false)[0]} />
```

Comment:
Now that we're able to work in bite-size pieces do you want to add unit tests for this in this PR?

Reply:
Great question. I don't think this component is currently setup well for a unit test. I'll address that in DAH-1122